.github/workflows/cd.yml

```
@@ -410,54 +75,54 @@
           method: "POST"
           customHeaders: |
             Content-Type: application/json
           data: |
             {
-              "text": ":broken_heart: 헬스 체크 실패 - `${{ github.repository }}` 브랜치 `${{ github.ref_name }}`"
+              "content": "❌ 헬스 체크 실패 - `${{ github.repository }}` 브랜치 `${{ github.ref_name }}`"
             }
 
-      - name: Notify Mattermost on Deploy Success
+      - name: Notify Discord on Deploy Success
         if: success() && steps.deploy-ec2.conclusion == 'success'
```

actionlint (1.7.7)

property "deploy-ec2" is not defined in object type {health-check: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)